bigquery_etl/alchemer/survey.py

```diff
@@ -52,22 +71,52 @@ def format_responses(s, date):
             data["options"] = list(data["options"].values())
         if data.get("subquestions"):
             data["subquestions"] = json.dumps(data["subquestions"])
-        results.append(data)
 
-    return {
-        # this is used as the partitioning field
-        "submission_date": date,
-        **{field: s[field] for field in fields},
-        "survey_data": results,
-    }
+        survey_data.append(data)
 
+    return survey_data
 
-def construct_data(survey, date):
-    """Construct response data."""
-    return [format_responses(resp, date) for resp in survey["data"]]
 
+def format_responses(
+    survey_response: dict, date: str, include_url_variables: bool
```

Comment:
I'd suggest having `include_url_variables` default to false in all these functions:
```suggestion
    survey_response: dict, date: str, include_url_variables: bool = False
```
That will also have the benefit of not needing to pass that argument to all the test calls of these functions except when you want to actually test that optional functionality by passing `include_url_variables=True`.